dev/archery/archery/benchmark/google.py

```diff
@@ -57,7 +57,7 @@ def results(self, repetitions=1, repetition_min_time=None):
                     "--benchmark_out_format=json"]
 
             if repetition_min_time is not None:
-                argv.append(f"--benchmark_min_time={repetition_min_time:.6f}")
+                argv.append(f"--benchmark_min_time={repetition_min_time:.6f}s")
```

Comment:
This was to suppress a message I saw in the logs:
```
LOG(0): Value passed to --benchmark_min_time should have a suffix. Eg., `30s` for 30-seconds.
```

Reply:
Yes, it's a change in the Google benchmark CLI.

Reply:
It seems that the warning was added with the flag itself https://github.com/google/benchmark/blame/70916cbf71f50b9e1e6f13559e10d6dbb92beb32/src/benchmark_runner.cc#L199 so this isn't an issue after all